### PR TITLE
Add Ploutos packaging workflow.

### PR DESCRIPTION
**Pull request by partim**

### CODE REVIEW

File: pkg/rules/packages-to-build.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    test-image: 'almalinux:9'
+
+  - pkg: 'dnsi'
+    image: 'rockylinux:9'
+    target: 'x86_64'
+    test-image: 'centos:9-Stream'
+
+test-mode:   
+  - 'fresh-install'
+  - 'upgrade-from-published'
```
Comment (ximon18) [inline pkg/rules/packages-to-build.yml:43]: upgrade-from-published should be removed as there is no prior version published to upgrade from.

File: pkg/rules/packages-to-build.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+test-exclude:
```
Comment (ximon18) [inline pkg/rules/packages-to-build.yml:45]: These exclusions can be removed.

File: .github/workflows/pkg.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      - v*
+
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  package:
+    uses: NLnetLabs/ploutos/.github/workflows/pkg-rust.yml@v7
+    secrets:
+      DOCKER_HUB_ID: ${{ secrets.DOCKER_HUB_ID }}
```
Comment (ximon18) [inline .github/workflows/pkg.yml:17]: TODO: The corresponding secrets for this (our Docker Hub automation user id and a newly generated access token granting access to a new dnsi Docker Hub repo) need to be provisioned in GH project settings.

File: .github/workflows/pkg.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    with:
+      docker_org: nlnetlabs
+      docker_repo: dnsi
+      docker_build_rules: pkg/rules/docker-images-to-build.yml
+      docker_sanity_check_command: --version
+
+      package_build_rules: pkg/rules/packages-to-build.yml
+      package_test_scripts_path: pkg/test-scripts/test-<package>.sh
+
+      rpm_scriptlets_path: pkg/rpm/scriptlets.toml
```
Comment (ximon18) [inline .github/workflows/pkg.yml:28]: The file referred to is not included in this PR.

File: .github/workflows/pkg.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+jobs:
+  package:
+    uses: NLnetLabs/ploutos/.github/workflows/pkg-rust.yml@v7
+    secrets:
+      DOCKER_HUB_ID: ${{ secrets.DOCKER_HUB_ID }}
+      DOCKER_HUB_TOKEN: ${{ secrets.DOCKER_HUB_TOKEN }}
+    with:
+      docker_org: nlnetlabs
+      docker_repo: dnsi
+      docker_build_rules: pkg/rules/docker-images-to-build.yml
```
Comment (ximon18) [inline .github/workflows/pkg.yml:22]: The file referred to is not included in this PR.

File: pkg/rules/packages-to-build.yml (hunk 1 above)
Comment (partim) [reply pkg/rules/packages-to-build.yml:43]: Ah, right. So I guess commenting it out is best.

**Review by ximon18 — APPROVED** [review]

**Pull request merged**